engine/internal/runtime/client.go

```diff
@@ -124,7 +128,12 @@ func (c *commonClient) deployRuntime(
 	update bool,
 ) (*appsv1.StatefulSet, error) {
 	mci := c.mconfig.ModelConfigItem(params.modelID)
-	name := resourceName(mci.RuntimeName, params.modelID)
+	var name string
+	if params.pullerDaemonMode {
+		name = resourceName(mci.RuntimeName, "dynamic")
```

Comment:
Slightly prefer defining a const for `dynamic` and use here and `ollama_client.go`.